rust/kcl-lib/src/std/extrude.rs

```diff
@@ -103,6 +103,15 @@ async fn inner_extrude(
 ) -> Result<Vec<Solid>, KclError> {
     let body_type = body_type.unwrap_or_default();
 
+    if matches!(body_type, BodyType::Solid)
+        && let Some(open_profile) = sketches.iter().find(|sk| matches!(sk.is_closed, ProfileClosed::No))
+    {
+        return Err(KclError::new_semantic(KclErrorDetails::new(
+            "Cannot solid extrude an open profile. Either close the profile, or use a surface extrude.".to_owned(),
+            open_profile.meta.iter().map(|m| m.source_range).collect(),
```

Comment:
The error is going to point to the profile, not where the `extrude()` call is? It's helpful to have both, but what we really want is to show additional location information the way rust-analyzer does. (We don't currently support that. If you add it to the array of source ranges, the frontend editor will label it as part of the backtrace.)

Given that we need to choose one, I would expect it to point to the `extrude()` since that's the line I could comment out which would make the error go away.